### PR TITLE
Add functionality for color name indicator

### PR DESCRIPTION
**Pull request by ThatChristianGuy**
I added the textlabel to the gui (sent to gigs on slack), this gives it functionality.

### CODE REVIEW

File: tools/Paint.lua [hunk 1]
```diff
@@ -198,6 +198,21 @@ Tools.Paint.showGUI = function ( self )
 				self:changeColor( BrickColor.new( ColorButton.Name ) );
 			end );
 		end;
+	
+		for i, v in pairs(Container.Palette:GetChildren()) do
```
Comment (GigsD4X) [inline tools/Paint.lua]: Can you be more descriptive with these variables? If `i` isn't being used, make sure to name it `_` :stuck_out_tongue:

File: tools/Paint.lua [hunk 2]
```diff
@@ -198,6 +198,21 @@ Tools.Paint.showGUI = function ( self )
...
+			if BrickColor.new(v.Name) then
+				v.MouseEnter:connect(function()
+					Container.Changes.Text.Text = v.Name;
+				end);
+				v.MouseLeave:connect(function()
+					if self.Options.Color then
+						Container.Changes.Text.Text = self.Options.Color.Name;
+					else
+						Container.Changes.Text.Text = 'select a color';
```
Comment (GigsD4X) [inline tools/Paint.lua]: Instead of 'select a color' let's just make it blank, so `''`

**Comment by GigsD4X** [comment]
Also, after those changes, can you copy/paste the new code into the actual scripts? Both in the tool version, and in the plugin version, then save those as .rbxmx files under the `build/` folder, and also make sure to test the changes to see if everything works as expected.

- [ ] Update tool & plugin objects
- [ ] Save as .rbxmx builds
- [ ] Test the tool and the plugin with the new changes
- [ ] Amend everything to this commit

**Comment by ThatChristianGuy** [comment]
How do I test the plugin? Just publish one without making it for sale? And do I publish the .rbxmx's or put the script here?

**Comment by GigsD4X** [comment]
Save it as an .rbxm under C:\Users\[Your Windows username]\AppData\Local\Roblox\Plugins. If the Plugins folder doesn't exist, you can create it & it should work. Just close/reopen Studio and it should load up, but make sure to disable the other one in Manage Plugins so that you don't have two & so that you know which one is which.

**Comment by GigsD4X** [comment]
Also, make sure to add a comment in the code describing what that chunk of code does, maybe right before it.

**Comment by GigsD4X** [comment]
Oh! And _in_ the build files, make sure they also include the changed GUIs.

**Comment by ThatChristianGuy** [comment]
Already sent the changed build files in slack, and I've been using my local versions, which have the new GUI :P

**Comment by ThatChristianGuy** [comment]
Moved to #82

**Pull request closed without merging**